docs/samples/git-setup/clustertemplates/version_4.Y/extra-manifest/01-container-mount-ns-and-kubelet-conf-master.yaml

```diff
@@ -0,0 +1,72 @@
+# Automatically generated by extra-manifests-builder
+# Do not make changes directly.
```

Comment:
Since they are directly taken from the ZTP container, they should not be changes for the validated DU profile.
For other profiles I still don't think they should be changed directly, but built from the cnf-features-deploy repo.
I think the main concern here would be to keep once source of truth.